app/assets/javascripts/application/messages.js

```diff
@@ -0,0 +1,47 @@
+var mumuki = mumuki || {};
+
+(function (mumuki) {
+    $(function () {
+        var token = new mumuki.CsrfToken();
+        setInterval(function () {
+            if ($('.badge-messages').length == 0) {
+                return;
+            }
+            $.ajax(token.newRequest({
+                url: '/messages',
+                type: 'GET'
+            })).done(function (res) {
+                $('.badge-messages').html(res.messages_count);
+                $('.messages-box').toggleClass('messages-box-empty', !res.has_messages);
+            }).fail(function (_error) {
+                //ignoring error, not important
+            });
+        }, 60000);
+    });
+
+
+}(mumuki));
+
+function submitMessagesForm(url) {
+    function success(data) {
+        $('#view-messages-template').html(data);
+        $('#messages-modal').modal();
+    }
+
+    function error(xhr) {
+        console.log(xhr);
```

Comment:
May be here, we should display an html with some text like this "Messages are not available right now. Try later"